BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -254,6 +261,22 @@ public class AppProcessor {
             services.errorReporter.log(error: error)
         }
     }
+
+    /// Starts timer to send organization events regularly
+    private func startEventTimer() {
+        sendEventTimer = Timer.scheduledTimer(withTimeInterval: 5, repeats: true) { _ in
```

Comment:
🎨 We don't ever deallocate `AppProcessor` so this is pretty minor, but I think the timer should be referencing a weak self here to avoid a retain cycle since the processor keeps a reference to the timer.

Reply:
It's still good housekeeping to do that, yeah. Good catch!